fiftyone/core/camera.py

```
@@ -0,0 +1,850 @@
+            points = (self._world_to_camera @ points_h.T).T[:, :3]
+
+        # Handle OpenGL convention (flip z and y)
+        if self.camera_convention == "opengl":
+            points = points.copy()
+            points[:, 1] = -points[:, 1]
+            points[:, 2] = -points[:, 2]
+
+        # Check for points behind camera
+        z = points[:, 2]
+        if np.any(z <= 0):
+            warnings.warn(
```

Ruff (0.14.10)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)